service/telemetry/generated_config.go

```diff
@@ -0,0 +1,59 @@
+// Code generated by github.com/atombender/go-jsonschema, DO NOT EDIT.
+
+package telemetry
+
+type Attributes struct {
+	// ServiceName corresponds to the JSON schema field "service.name".
+	ServiceName *string `json:"service.name,omitempty" yaml:"service.name,omitempty" mapstructure:"service.name,omitempty"`
```

Comment:
Can we configure the tool to only set the mapstructure tags?

Reply:
The tool doesn't support this at the moment: https://github.com/omissis/go-jsonschema/blob/172061399cba4a93f95ac5cf3b3935daa147e966/pkg/generator/generate.go#L807-L811

Reply:
Added a feature request here: https://github.com/omissis/go-jsonschema/issues/79

Reply:
hi @bogdandrutu , go-jsonschema mantainer here :) I was wondering if the extra tags are causing any trouble, or if it's just for "cosmetic" reasons, eg: for maintaining a consistent code style or something. Thanks!

Reply:
It is extra public API for us that we don't want to expose. Once exposed others may rely on them, and I don't want to maintain these.

Reply:
for now i've updated gojsonschema to use the updated version in my fork.

Reply:
hi @codeboten sorry for the delay! I reviewed and merged the PR, and release a new v0.12.0 as well :)